src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -145,10 +145,26 @@ public void addAuthor(Author author, List<String> ignoreGlobList) {
     public void removeIgnoredAuthors(List<String> ignoredAuthorsList) {
         for (String author : ignoredAuthorsList) {
             if (authorEmailsAndAliasesMap.containsKey(author)) {
-                authorList.remove(authorEmailsAndAliasesMap.get(author));
+                removeIgnoredAuthorsInformation(author);
             }
         }
-        resetAuthorInformation();
+    }
+
+    /**
+     * Removes all information of the {@code author} from the configs
+     */
+    public void removeIgnoredAuthorsInformation(String author) {
+        Author ignoredAuthor = authorEmailsAndAliasesMap.get(author);
+        authorList.remove(ignoredAuthor);
+        authorDisplayNameMap.remove(ignoredAuthor);
+        authorEmailsAndAliasesMap.remove(author);
+        authorEmailsAndAliasesMap.remove(ignoredAuthor.getGitId());
+
+        List<String> aliases = new ArrayList<>(ignoredAuthor.getAuthorAliases());
```

Comment:
a clarification: why do we need to create a new arraylist?

Reply:
I felt the code would be more readable if we create an `aliases` list first. And then use the list. Instead of working directly by calling it.

Reply:
I see. Doesn't `getAuthorAliases` return a list though?